subsys/bluetooth/controller/ll_sw/nordic/lll/lll.c

```diff
@@ -340,11 +341,25 @@ u32_t lll_evt_offset_get(struct evt_hdr *evt)
 u32_t lll_preempt_calc(struct evt_hdr *evt, u8_t ticker_id,
 		       u32_t ticks_at_event)
 {
-	/* TODO: */
+	u32_t ticks_now = ticker_ticks_now_get();
+	u32_t diff;
+
+	diff = ticker_ticks_diff_get(ticks_now, ticks_at_event);
+	diff += 3;
```

Comment:
a small comment here about this hardcoded value?

Reply:
Help me here with a define name; 3 is the number of ticks between the high watermark and full tank. i.e. if the actual ticks plus 3 ticks exceeds the hard real-time deadline tick.

Reply:
`SAFETY_TICKS` or `TICKS_TO_DEADLINE`

Reply:
@carlescufi the right macro to use is HAL_TICKER_CNTR_CMP_OFFSET_MIN, hence updated now.